extensions/gc/tasks/GarbageCollectorTask.js

```diff
@@ -44,6 +44,14 @@ class GarbageCollectorTask extends BackbeatTask {
                 size: location.size,
                 dataStoreVersionId: location.dataStoreVersionId,
             })),
+            IfUnmodifiedSince: entry.getAttribute('source.lastModified'),
+            Bucket: entry.getAttribute('source.bucket'),
+            Key: entry.getAttribute('source.objectKey'),
+            StorageClass: entry.getAttribute('source.storageClass'),
+            Tags: JSON.stringify({
+                'scal-delete-marker': 'true',
+                'scal-delete-service': 'lifecycle',
```

Comment:
I don't think we know it comes from lifecycle at this point. It would have to be passed as a separate attribute in the action message IMO (like an optional `"origin": "lifecycle"` attribute for example)

minor point: I would slightly prefer a naming such as `scal-delete-marker-created-by` or so, which looks more precise to me.

Reply:
Nice catch. Will follow with that change.